src/json-output.ts

```diff
@@ -58,12 +66,18 @@ export function jsonOutput(results: ResultStatsWithDifferences[]):
     }
     benchmarks.push({
       name: result.result.name,
-      samples: result.result.millis,
+      bytesSent: result.result.bytesSent,
+      version: result.result.version ? result.result.version : undefined,
```

Comment:
If `version` is falsey (particularly empty string if there is no version for this run), I'm setting it to `undefined` here so that when this object is passed `JSON.stringify` and written to a file, the field is dropped.